src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/typer.ts

```diff
@@ -8,4 +10,13 @@ export interface ITidligereInnvilgetVedtak {
     harTidligereOvergangsstønad: boolean;
     harTidligereBarnetilsyn: boolean;
     harTidligereSkolepenger: boolean;
+    periodeHistorikkOvergangsstønad: IGrunnlagsdataPeriodeHistorikk[];
+}
+
+export interface IGrunnlagsdataPeriodeHistorikk {
+    antMnd: number;
+    antallMndUtenBeløp: number;
```

Comment:
```suggestion
    antallMåneder: number;
    antallMånederUtenBeløp: number;
```

Vi har pleid å skrive disse ordene helt ut ellers i løsningen